source/sync/configure/sync-settings.txt

```diff
@@ -267,47 +264,21 @@ Sync (recommended). If you are using the older Partition-Based Sync, refer to
      "state": <"enabled" | "disabled">,
      "client_max_offline_days": <Number>,
      "is_recovery_mode_disabled": <Boolean>,
-     "permissions": {
-       "rules": {
-         "<Type Name>": [
-           {
-             "name": <String>,
-             "applyWhen": { <Expression> },
-             "read": <Expression>,
-             "write": <Expression>,
-             "fields": {
-               "<Field Name>": {
-                 "read": <Boolean>,
-                 "write": <Boolean>,
-                 "fields": <Embedded Object Fields>
-               }
-             }
-             "additional_fields": {
-               "read": <Boolean>,
-               "write": <Boolean>
-             }
-           }
-         ]
-       },
-       "defaultRoles": [
-         {
-           "name": <String>,
-           "applyWhen": { <Expression> },
-           "read": <Expression>,
-           "write": <Expression>
-         }
-       ]
-     },
      "queryable_fields_names": [
        <Array of String Field Names>
-     ]
+     ],
+     "permissions": "<Deprecated, Do Not Use>"
    }
 
-Root Object
-~~~~~~~~~~~
+Note: the ``permissions`` field is no longer used, but it might still appear in
```

Comment:
The field will continue to be used in unmigrated apps, however; On second thought, I think it might be worth holding off from making this change until all apps are migrated (to make sure people aren't accidentally deleting their entire permissions configuration if their app isn't migrated yet)

Reply:
Do you have a link to an announcement or post about the migration? It might be handy to put a reference here in this case.

Reply:
One thing worth calling out there is that we're planning on exposing the feature _only for new apps_ tomorrow. We'll likely let it bake for roughly a week before attempting to migrate any existing apps (ie, the migration of existing apps into the unified rules system will happen over the subsequent week or two). Not sure what the best way to communicate this is, but might be worth holding back from mentioning a specific date?